src/include/wiredtiger.in

```diff
@@ -1763,7 +1763,7 @@ struct __wt_session {
 	 * A transaction must be in progress when this method is called.
 	 *
 	 * If WT_SESSION::commit_transaction returns an error, the transaction
-	 * was rolled back, not committed.
+	 * was rolled back, not committed, and all cursors are reset.
```

Comment:
Probably worth clarifying this is not "all cursors in the application", and checking to be sure that error isn't repeated elsewhere.

Reply:
[933e24f](https://github.com/wiredtiger/wiredtiger/pull/8027/commits/933e24f98dd60e3134046b7ff3fe654b4af01164) updates this (and two other examples) to use the same language we use elsewhere, "All cursors associated with the session are reset."